common/c/targets/esp8266/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+#
+#  Copyright 2014-2016 CyberVision, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+cmake_minimum_required(VERSION 2.8)
+
+if(NOT DEFINED ESPRESSIF_HOME)
+    set(ESPRESSIF_HOME /opt/Espressif)
+endif()
+
+set(ESP_SDK_HOME ${ESPRESSIF_HOME}/esp-rtos-sdk)
```

Comment:
`esp-rtos-sdk` can be in a totally different location (not in `esp-rtos-sdk` directory). Replace `ESPRESSIF_HOME` with `ESP_RTOS_SDK` and make user specify his directory.